src/compiler.jl

```diff
@@ -896,54 +900,55 @@ function array_shadow_handler(B::LLVM.API.LLVMBuilderRef, OrigCI::LLVM.API.LLVMV
     b = LLVM.IRBuilder(B)
     orig = LLVM.Value(OrigCI)
 
-    vals = LLVM.Value[]
```

Comment:
Can we restore the indentation here? It'll both look prettier and also reduce the size of the diff